test/functional-tests/dunstrc.progress_bar

```diff
@@ -0,0 +1,10 @@
+[global]
+    font = Monospace 8
+    allow_markup = yes
+    format = "<b>%s</b>\n%b"
+    geometry = "0x5-30+20"
+	icon_position = left
```

Comment:
There's some inconsistent indentation here, tabs instead of space.

Reply:
Got it. I set my vimrc to tabexpand to prevent this in the future.